app/models/guides_filter.rb

```diff
@@ -0,0 +1,55 @@
+class GuidesFilter
+  VALID_FILTERS = [
+    'author',
+    'content_owner',
+    'page',
+    'page_type',
+    'q',
+    'state'
+  ]
+
+  def initialize(scope)
+    @scope = scope
+    # TODO: :content_owner not being included is resulting in an N+1 query
```

Comment:
I just haven't got round to fixing it yet.. content_owner joins `guides` and if the user is searching, using the query in `Guide.search`, then the reference to `tsv` is ambiguous. You end up returning guides because the search matched it's content_owner.

It just needs to be preloaded separately..